src/PrestaShopBundle/Component/ActionBar/ActionsBarButton.php

```diff
@@ -26,56 +26,11 @@
 
 namespace PrestaShopBundle\Component\ActionBar;
 
-class ActionsBarButton implements ActionsBarButtonInterface
-{
-    /**
-     * @var string
-     */
-    protected $class;
-
-    /**
-     * @var string[]
-     */
-    protected $properties;
-
-    /**
-     * @var string
-     */
-    protected $content;
-
-    /**
-     * @param string $class
-     * @param string[] $properties
-     * @param string $content
-     */
-    public function __construct(string $class = '', array $properties = [], string $content = '')
-    {
-        $this->class = $class;
-        $this->properties = $properties;
-        $this->content = $content;
-    }
-
-    /**
-     * @return string
-     */
-    public function getClass(): string
-    {
-        return $this->class;
-    }
+use PrestaShop\PrestaShop\Core\Action\ActionsBarButton as CoreActionsBarButton;
```

Comment:
Sorry, @sowbiba I'm gonna be a PITA again 😅

I didn't see @atomiix comment but yes it was on purpose that this file was moved back to src/PrestaShopBundle/Controller/Admin/Sell/Order/ActionsBarButton.php

Although it's far from intuitive when you see this PR only so I'll detail the history of this class:
- it was first created in 177 for [order page only](https://github.com/PrestaShop/PrestaShop/pull/16144/files#diff-9b819489a3b36887d93d6e17e127dd4675769384b08a5eeb6c584add2b05e5f7R29) 
- then it was moved during 8.0 development in the [bundle namespace](https://github.com/PrestaShop/PrestaShop/pull/25066/files#diff-cb6babcba28e0256be1189d4053432b66172fa28b7f731363bc6f90cd7c93455R27)  which allowed to use them for more generic use cases but also created a breaking change that could be avoided with temporary class extension
- finally we decided in this PR that this class was not really fit for the bundle namespace (since it has no link with Symfony) but rather should be in the core in the `PrestaShop\PrestaShop\Core\Action`

So the idea here is to fix two things:
- move those classes to the proper namespace
- reintroduce deleted classes from 178 and make them extend the new ones
- we can do this before FF for 8.0 thus avoiding introducing a BC break and improving the architecture

Reply:
TLDR; those files should be moved into `src/PrestaShopBundle/Controller/Admin/Sell/Order` folder and `PrestaShopBundle\Controller\Admin\Sell\Order` namespace to match their initial place from 1.7 versions

Reply:
sorry I forgot the history, I looked at the @Progi1984  PR and approve the feedback of @atomiix 🤦🏾‍♂️